cylc/flow/task_job_mgr.py

```diff
@@ -645,25 +664,41 @@ def _run_job_cmd(self, cmd_key, suite, itasks, callback):
         Group itasks with their user@host.
         Put a job command for each user@host to the multiprocess pool.
 
+        n.b. When doing tasks like kill we may want to use the same host.
+        so we get the host and owner back from the itask and keep this working
+        using hosts rather than platforms.
```

Comment:
Can you jog my memory on this one, I can't quite remember why.

Reply:
This is outdated and no-longer in the code base. 
I don't think it's actually true.